yateto/gemm_configuration.py

```diff
@@ -240,18 +286,19 @@ class DefaultGeneratorCollection(GeneratorCollection):
   def __init__(self, arch):
     super().__init__([])
     libxsmm = LIBXSMM(arch)
+    libxsmm_jit = LIBXSMM_JIT(arch)
     pspamm = PSpaMM(arch)
     mkl = MKL(arch)
     blis = BLIS(arch)
     openblas = OpenBLAS(arch)
     eigen = Eigen(arch)
     forge = GemmForge(arch)
     defaults = {
-      'snb' : [libxsmm, mkl, blis, eigen],
-      'hsw' : [libxsmm, mkl, blis, eigen],
-      'rome' : [libxsmm, blis, eigen],
-      'knl' : [libxsmm, pspamm, mkl, blis, eigen],
-      'skx' : [libxsmm, pspamm, mkl, blis, eigen],
+      'snb' : [libxsmm_jit, libxsmm, mkl, blis, eigen],
+      'hsw' : [libxsmm_jit, libxsmm, mkl, blis, eigen],
+      'rome' : [libxsmm_jit, libxsmm, blis, eigen],
+      'knl' : [libxsmm_jit, libxsmm, pspamm, mkl, blis, eigen],
+      'skx' : [libxsmm_jit, libxsmm, pspamm, mkl, blis, eigen],
       'thunderx2t99' : [pspamm, openblas, blis, eigen],
```

Comment:
btw, do you want to add `libxsmm_jit` here?

Reply:
Yes, after testing :)